src/Markdig.Tests/TestLinkHelper.cs

```diff
@@ -31,6 +31,16 @@ public void TestUrlUrl()
         }
 
         [Test]
+        public void TestUrlTrailingFullStop()
+        {
+            var text = new StringSlice("http://google.com.");
```

Comment:
Could you add another case with the space character at the end? (can be in the same test method)... even if the code checking for the end is shared with IsEndOfUri...